layouts/shortcodes/csv-block-table.html

```diff
@@ -13,6 +14,7 @@
       <tr>
         <td>network</td>
         <td>IP network as a string</td>
+        {{ if eq $includeUtility "true" }}
```

Comment:
You could avoid repeating the shared text with something like this:

```handlebars
<td>
  This is the IPv4 or IPv6 network in CIDR format such as
  &quot;2.21.92.0/29&quot; or &quot;2001:4b0::/80&quot;.
  {{ if eq $includeUtility "true" }}
    <span>
      We offer a
      utility to convert this column to start/end IPs or start/end
      integers. See
      <a
        href="#conversion-utility"
      >
        the conversion utility
        section
      </a>
      for details.
    </span>
  {{ end }}
</td>
```
I think that is easier to read/easier to tell the difference between the two branches/easier to update if there is a copy change in the future. The `<span>` is optional.